docs/source/topics/database.rst

```diff
@@ -222,12 +222,18 @@ List of all relationships:
 +------------------+---------------+--------------------+-------------------------------------------------+
 
 .. _topics:database:advancedquery:queryhelp:
+.. _topics:database:advancedquery:querydict:
 
-The queryhelp
--------------
+Converting the QueryBuilder to/from a dictionary
```

Comment:
This should be inserted as the new reference for the link in the REST API help...

Reply:
done